src/components/VideoPlayerContexts/types.ts

```diff
@@ -0,0 +1,42 @@
+import type {Video} from 'expo-av';
+import type {MutableRefObject} from 'react';
+import type {View} from 'react-native';
+import type {SharedValue} from 'react-native-reanimated';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type PlaybackContext = {
+    updateCurrentlyPlayingURL: (url: string) => void;
+    currentlyPlayingURL: string | null;
+    originalParent: View | null;
+    sharedElement: View | null;
+    currentVideoPlayerRef: MutableRefObject<Video | null>;
+    shareVideoPlayerElements: (ref: Video, parent: View, child: View, isUploading: boolean) => void;
+    playVideo: () => void;
+    pauseVideo: () => void;
+    checkVideoPlaying: (statusCallback: (isPlaying: boolean) => void) => void;
+};
+
+type VolumeContext = {
+    updateVolume: (newVolume: number) => void;
+    volume: SharedValue<number>;
+};
+
+type SingularMenuItem = {
+    icon: IconAsset | null;
+    text: string;
+    onSelected: () => void;
+    shouldPutLeftPaddingWhenNoIcon?: boolean;
+};
+
+type MenuItem = {
+    icon: IconAsset;
+    text: string;
+    subMenuItems: SingularMenuItem[];
+};
+
+type VideoPopoverMenuContext = {
+    menuItems: Array<SingularMenuItem | MenuItem>;
```

Comment:
I think it should be possible to reuse [PopoverMenuItem](https://github.com/Expensify/App/blob/main/src/components/PopoverMenu.tsx#L19) type, cause menuItems are [passed](https://github.com/Expensify/App/blob/main/src/components/VideoPopoverMenu/index.js#L34) to PopoverMenu then.
```suggestion
    menuItems: PopoverMenuItem[];
```

Reply:
New types have slightly different props - SingularMenuItem icon might be null and MenuItem does not have onSelected. Both icon and onSelected are required for PopoverMenuItem. Unless we make these props optional in PopoverMenu, it is not possible to reuse this type.

Reply:
But these types should be aligned anyway, otherwise I think there will be TS problems during [VideoPopoverMenu](https://github.com/Expensify/App/blob/main/src/components/VideoPopoverMenu/index.js) migration. But if you think it's extra in this PR we can make it be resolved in VideoPopoverMenu migration.
 ^^ @blazejkustra

Reply:
@VickyStash is right! Let's adjust the type in this PR